aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -100,6 +95,8 @@ module aptos_framework::stake {
     /// 2. If validator calls unlock, their stake is moved directly to inactive.
     /// 3. When the next epoch starts, the validator can be activated if their active stake is more than the minimum.
     struct StakePool has key {
+        // This may be instantiated without a valid validator config
+        valid_validator_info: bool,
```

Comment:
Instead of having this bool, can we just check the existence of ValidatorConfig and that the addresses are not empty vectors?

Reply:
I'd prefer either this or an option, leaving this to evaluate the contents is tricky and becomes inconsistent.